DB/projects.json

```
@@ -3556,4 +3556,15 @@
       }
     ]
   }
+  {
+    "github_username": "DarhkVoyd",
+    "Projects": [
+      {
+        "github_url": "https://github.com/enrich-platforms/toppings",
+        "title": "Toppings",
+        "description": "Toppings adds an extra layer of flavor for a more efficient and convenient web.",
+        "tech": ["javascript", "typescript", "nodejs"]
+      },
```

remove the last comma
```suggestion
      }
```